modules/installation-azure-marketplace.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-account.adoc
+
+:_content-type: CONCEPT
+[id="installation-azure-marketplace_{context}"]
+= Supported Azure Marketplace regions
+
+Installing a cluster using the Azure Marketplace image is available to customers who purchase the offer in North America and EMEA.
+
+While the offer must be purchased in North America or EMEA, you can deploy the cluster to any of the Azure public partitions that {product-title} supports.
+
+[NOTE]
+====
+Deploying a cluster using the Azure Marketplace image is not supported for the Azure Government regions.
```

Comment:
```suggestion
Deploying a cluster by using the Azure Marketplace image is not supported for the Azure Government regions.
```

Reply:
@bscott-rh I added this item for the 4.8 version after you made the initial changes.

Reply:
Do you think I should also change line 9 from 
`Installing a cluster using the Azure Marketplace image is available to customers who purchase the offer in North America and EMEA.`
to 
`Installing a cluster by using the Azure Marketplace image is available to customers who purchase the offer in North America and EMEA.`

for consistency with this change?

Reply:
@bscott-rh  Yes, that change is correct and consistent with the others. Thanks!